allure-report-face/src/index.js

```diff
@@ -0,0 +1,19 @@
+import App from './app';
+import $ from 'jquery';
+import Backbone from 'backbone';
+import allurePlugins from './pluginApi';
+import './plugins/default';
+import './plugins/defects';
+import './plugins/environment';
+import './plugins/xunit';
+import './plugins/behaviors';
+import './plugins/graph';
+import './plugins/timeline';
+
+import DescriptionView from './plugins/testcase-description/DescriptionView';
+allurePlugins.addTestcaseBlock(DescriptionView, {position: 'before'});
+
+App.start();
+Backbone.history.start();
+
+window.jQuery = $;
```

Comment:
It was needed me for debugging.

Anyway, till release I am going to expose `Backbone`, `Underscore` and `Handlebars` as globals too. It the only way how to reuse them in plugins.